torchinductor/ir.py

```diff
@@ -2840,6 +2840,49 @@ def __init__(self, layout, input, index):
     def should_allocate(self):
         return False
 
+class LinearEltwise(ExternKernelAlloc):
+    kernel = "torch.ops.mkldnn_prepacked.linear_eltwise"
```

Comment:
About the naming: Is the post-op unary only? Also, to be consistent with existing IR names, would "pointwise" be better than "eltwise"?